src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -201,9 +202,10 @@ public EmailWrapper generateFeedbackSubmissionConfirmationEmailForInstructor(
             String template, String subject) {
 
         List<EmailWrapper> emails = new ArrayList<EmailWrapper>();
+        String coOwnersLine = generateCoOwnersEmailsLine(generateCoOwnersList(course.getId()));
         for (InstructorAttributes instructor : instructors) {
             emails.add(generateFeedbackSessionEmailBaseForInstructorReminders(course, session, instructor,
-                                                                              template, subject));
+                                                                              template, subject, coOwnersLine));
```

Comment:
Fixed in https://github.com/TEAMMATES/teammates/pull/7169/commits/d09473731b8a57453e131701ac0557a89c8e0b8d